### PR TITLE
Doc final updates

### PR DESCRIPTION
**Pull request by praveenraj2018**
Minor updates:
- Updated tool versions in the README.md
- Removed unwanted files
- Commented out the STAR version check (somehow it was missed in the last code merge)

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnavar/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnavar _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,19 +25,20 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
+9. Scatter one interval-list into many interval-files ([`GATK v4.2.5.0 IntervalListTools`](https://gatk.broadinstitute.org/hc/en-us/articles/4409917392155-IntervalListTools-Picard-))
+10. Call SNPs and indels ([`GATK v4.2.5.0 HaplotypeCaller`](https://gatk.broadinstitute.org/hc/en-us/articles/4409897180827-HaplotypeCaller))
+11. Merge multiple VCF files into one VCF ([`GATK v4.2.5.0 MergeVCFs`](https://gatk.broadinstitute.org/hc/en-us/articles/4409924817691-MergeVcfs-Picard-))
+12. Index the VCF ([`GATK v4.2.5.0 IndexFeatureFile`](https://gatk.broadinstitute.org/hc/en-us/articles/4409917499931-IndexFeatureFile))
+13. Filter variant calls based on certain criteria ([`GATK v4.2.5.0 VariantFiltration`](https://gatk.broadinstitute.org/hc/en-us/articles/4409897204763-VariantFiltration))
+14. Annotate variants ([`snpEff v5.0`](https://pcingola.github.io/SnpEff/se_introduction/), [Ensembl VEP v104.3](https://www.ensembl.org/info/docs/tools/vep/index.html))
+15. Present QC for raw read, alignment, gene biotype, sample similarity, and strand-specificity checks ([`MultiQC v1.12`](http://multiqc.info/), [`R`](https://www.r-project.org/))
```
Comment (maxulysse) [inline README.md:41]: I don't think having the tools version here is good for easy visibility.
Maybe adding it in a separate table?

Comment (praveenraj2018) [reply README.md:41]: I thought about it earlier. Yes, a table is good. Do you prefer to have it as a markdown table or an image?

Comment (maxulysse) [reply README.md:41]: md, that would be much easier to keep up to date

**Review by maxulysse — APPROVED** [review]
<3

**Pull request merged**